### ISSUE TITLE
Main level option not used in streams (v1.0.1)

### ISSUE THREAD

**Issue by eabay** [trentm/node-bunyan#164]
`level` option was used in all streams until 1.0.1 version. Now you have to define it for each stream, otherwise default level used no matter what passed in `bunyan.createLogger`

Is this intentional or a bug? It's not in the changelog.

**Comment by trentm** [comment]
@eabay Sorry for the delay in responding. By your description it *sounds* like a bug, but I don't know exactly what you mean yet.

**Comment by trentm** [comment]
Guessing the change in behaviour is from commit 334e36cdfd:

```
commit 334e36cdfd5683e3f26c023e70a866231ebeabaf
Author: Andrei Neculau <andrei.neculau@gmail.com>
Date:   Thu Apr 17 14:37:44 2014 +0200

    fix #105: export addStream and addSerializers
```

**Comment by trentm** [comment]
Oh, I see the bug. Yikes, that sucks! I'll try to fix this tonight.

**Comment by trentm** [comment]
Bad bug. This'll be released in version 1.3.1 soon (probably tonight).  Thanks for reporting this!

**Comment by eabay** [comment]
@trentm Thank you for the fix and thank you for bunyan!
